functional_testing/zrobot_templates/framework/zos_utils/utils.py

```diff
@@ -38,3 +41,62 @@ def create_container(self, **kwargs):
 
     def create_vm(self, **kwargs):
         return self.handle_blueprint('vm.yaml', **kwargs)
+
+    def check_vnc(self, vnc_ip):
+        vnc = 'vncdotool -s %s' % vnc_ip
+        result,error = self.execute_shell_commands(cmd="%s type %s key enter" % (vnc, repr('ls')))       
+        return error 
+        
+    def enable_ssh_access(self, vnc_ip, username=None, password=None):
+
+        username = username or self.vm_username
+        password = password or self.vm_password
+
+        """
```

Comment:
Please add the doc string at the beginning of the function

Reply:
done